VERSION

```diff
@@ -1 +1 @@
-1.0.2
+1.1.0
```

Comment:
@galexrt Do I need to increase this version if the only changes are in helm?

Reply:
@zalsader No, a version change of the "software" is not needed when only the Helm chart is changed.